backend/util.py

```diff
@@ -0,0 +1,24 @@
+import functools
+
+from flask_discord import DiscordOAuth2Session, requires_authorization, current_app, exceptions as disc_exceptions
+#from flask import exceptions
+
+class MissingMembership(disc_exceptions.HttpException):
+    "Exception when required guild membership is missing."
+
+def user_is_member_of(guild_id):
+    user = current_app.discord.fetch_user()
+    guilds = current_app.discord.fetch_guilds()
+    return any(g.id == guild_id for g in guilds)
+
+def requires_membership(guild_id):
+    def decorator(view):
+        @functools.wraps(view)
+        def wrapper(*args, **kwargs):
```

Comment:
This is where the meat of this change is. I copied this from discord_flask. Maybe I should add a comment to that effect?